full-stack-tests/imodelhub-client/src/imodelhub/Versions.test.ts

```diff
@@ -127,7 +127,7 @@ describe("iModelHub VersionHandler", () => {
       await utils.deleteIModelByName(requestContext, contextId, imodelName);
       await utils.deleteIModelByName(requestContext, contextId, imodelName2);
       utils.getRequestBehaviorOptionsHandler().resetDefaultBehaviorOptions();
-      iModelClient.requestOptions.setCustomOptions(utils.getRequestBehaviorOptionsHandler().toCustomRequestOptions());
+      iModelClient?.requestOptions.setCustomOptions(utils.getRequestBehaviorOptionsHandler().toCustomRequestOptions());
```

Comment:
Why is the optional property access operator here? can iModelClient ever be undefined?

Reply:
Yes, it is undefined when iModel creation timeouts on "before" step